conf/zeppelin-env.cmd.template

```diff
@@ -39,7 +39,9 @@ REM set ZEPPELIN_IDENT_STRING   		REM A string representing this instance of zep
 REM set ZEPPELIN_NICENESS       		REM The scheduling priority for daemons. Defaults to 0.
 REM set ZEPPELIN_INTERPRETER_LOCALREPO         REM Local repository for interpreter's additional dependency loading
 REM set ZEPPELIN_INTERPRETER_DEP_MVNREPO       REM Maven principal repository for interpreter's additional dependency loading
+REM set ZEPPELIN_HELIUM_NODE_INSTALLER_URL     REM Remote Node installer url for Helium dependency loader
 REM set ZEPPELIN_HELIUM_NPM_REGISTRY           REM Remote Npm registry for Helium dependency loader
```

Comment:
I avoided changing this one as it will "break" releases already using ZEPPELIN_HELIUM_NPM_REGISTRY (config needs to be manually adjusted). 
But happy to rename it. Is there anything "special" I need to do to inform community that they need to rename ZEPPELIN_HELIUM_NPM_REGISTRY to ZEPPELIN_HELIUM_NPM_INSTALLER_URL? Thanks.